GitCommands/Git/GitRevision.cs

```diff
@@ -86,6 +83,76 @@ public BuildInfo BuildStatus
         public string Subject { get; set; } = "";
         [CanBeNull]
         public string Body { get; set; }
+
+        [CanBeNull]
+        public string BodySummary
+        {
+            get
+            {
+                if (string.IsNullOrWhiteSpace(Body) || !HasMultiLineMessage)
+                {
+                    return Body;
+                }
+
+                return BuildSummary(Body.TrimEnd());
+            }
+        }
+
+        private static string BuildSummary(string body)
+        {
+            var s = new StringBuilder(100);
+
+            int lineCount = 0;
+            int lineStartPos = 0;
+            int pos;
+            for (pos = 0; pos < body.Length; ++pos)
+            {
+                if (body[pos] == '\n')
+                {
+                    if (pos - lineStartPos > CommitSummaryMaxLineLength)
+                    {
+                        AppendLine(CommitSummaryMaxLineLength, withEllipsis: true);
+                    }
+                    else
+                    {
+                        AppendLine(pos - lineStartPos);
+                    }
+
+                    lineStartPos = pos + 1;
+
+                    if (++lineCount == CommitSummaryMaxNumberOfLines)
+                    {
+                        return s.Append("[...]").ToString();
```

Comment:
This should be ellipsis …
(similar in other parts)
Inconsistent use in GE though

Reply:
That's what I have done first but I didn't found it enough clear. Even when developing the feature and didn't understood immediately that it was added by GE.
Maybe because the user could also use  '..;` in it's commit message.

I could change it because that's not a big deal for me but before that I would like the point of view of others... /cc @RussKie @mstv @drewnoakes @Balucsek

Reply:
> This should be ellipsis …

Could you please point to examples?



> I would like the point of view of others...

If we do so in other parts (especially in high traffic dialogs), we should be consistent

Reply:
I agree to use `…` for consistency. Though I am afraid that this as a single glyph would be hard to notice.
"[…]", although actually "[...]" is more similar to the multi-line body indicator ![grafik](https://user-images.githubusercontent.com/36601201/68511360-116d6580-0276-11ea-93df-c7baf14af497.png) in the message column of the revision grid.

Reply:
… was used in CommitInfo until recently (there are remains checking for … in LinkFactory)
It is used in the repository list still

Other shortening is using three dots

There was a discussion about this a year ago or so

Reply:
In that case, it probably doesn't matter.
I don't have a strong preference, so let's go with whatever we have implemented. We can unify to one format later.